src/java.base/share/native/libzip/zip_util.c

```diff
@@ -809,7 +810,7 @@ ZIP_Get_From_Cache(const char *name, char **pmsg, jlong lastModified)
 
     if (strlen(name) >= PATH_MAX) {
         if (pmsg) {
```

Comment:
Some pre-existing references to pmsg still use implicit comparison, though not material to this PR, can you update them to compare with NULL.

Reply:
done.